alerts/UA00002.json

```diff
@@ -1,9 +1,11 @@
 {
   "dateRaised": 20201028,
-  "affectedFirmware": ["all"],
+  "affectedMasterFirmware": ["all"],
+  "affectedStableFirmware": ["copter", "plane"],
   "hardwareLimited": ["CUAVv5", "CubeBlack", "CubeBlack+", "CubeGreen-solo", "CubeOrange", "CubePurple", "CubeSolo", "CubeYellow", "DrotekP3Pro", "Durandal", "fmuv2", "fmuv3", "fmuv5", "mRoX21", "mRoX21-777", "Pix32v5", "Pixhawk1", "Pixhawk1-1M", "Pixhawk4"],
   "description": "When using SBUS output, if the RC receiver is lost the IOMCU will invert the SBUS output momentarily whilst searching for an RC input. This can lead to a loss of signal on the SBUS output port. This only affects boards with an IOMCU that use the default SBUS Output port.",
-  "mitigation": "Do not use the default SBUS OUT port. Use an alternative UART for SBUS Output.",
+  "preReleaseMitigation": "Do not use the default SBUS OUT port. Use an alternative UART for SBUS Output.",
```

Comment:
Sorry to be picky but I don't think adding a new field is helpful.  This creates another field that the developers or release manager needs to fill in and the GCS isn't going to know which to display so it will just display both so we might as well just have a single field.  Most users also aren't going to understand the difference between the fields.

Why don't we just make the (single) mitigation, "Upgrade to ArduCopter 4.0.5 (or later) or ArduPlane 4.0.7 (or later).  If the user must stay on Copter-4.0.4 or Plane-4.0.6 then do no use the default SBUS Out port, use an alternative UART for SBUS output"